repository/t.json

```diff
@@ -327,6 +327,18 @@
 				}
 			]
 		},
+		{
+			"name": "TeaCode Helper",
+			"details": "https://github.com/Apptorium/TeaCode-Sublime-Helper",
+			"labels": ["teacode"],
```

Comment:
Since this label is in the package name, and there is unlikely to be another teacode package,  it can be removed